crates/burn-vision/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+//! Vision ops for burn, with GPU acceleration where possible.
+//!
+//! # Operations
+//! Operation names are based on `opencv` wherever applicable.
+//!
+//! Currently implemented are:
+//! - `connected_components`
+//! - `connected_components_with_stats`
+//!
+
+#![warn(missing_docs)]
+
+extern crate alloc;
+
+/// Backend implementations for JIT and CPU
+pub mod backends;
+mod ops;
+mod tensor;
+
+#[cfg(feature = "export-tests")]
+#[allow(missing_docs)]
```

Comment:
I don't think we need to export the tests, since they run in this crate

Reply:
Integration tests count as running in a separate crate, and we need to use integration tests because of annoying rust limitations around `macro_export` (which is generated by `burn-tensor-testgen`).

Reply:
I don't think we're using `burn-tensor-testgen` in `burn-core`, and yeah we're probably not running the tests as integration tests for that reason.

Reply:
We might want to parameterize some of the tests at some point, so I think using testgen right away is better. I just didn't add a parameterized test yet because connected components are almost never used with anything other than u32. I believe opencv also supports u16 but I don't think it's work adding parameterized tests until there's more ops. It's good to have the infrastructure for it regardless.